src/main.js

```diff
@@ -0,0 +1,114 @@
+const myLibrary = [];
+localStorage.setItem('items', JSON.stringify(myLibrary));
+
+
+function Book(author, title, pages, read) {
+  this.author = author;
+  this.title = title;
+  this.pages = pages;
+  this.read = read;
+}
+
+function setStatus(read, button) {
+  if (read) {
+    button.innerText = 'Read';
+  } else {
+    button.innerText = 'Not read yet';
+  }
+}
+
+function switchStatus(e) {
+  const index = e.srcElement.attributes['1'].value;
+  myLibrary[parseInt(index, 10)].read = !myLibrary[parseInt(index, 10)].read;
+  setStatus(myLibrary[parseInt(index, 10)].read, e.target);
+}
+
+function removeBook(e) {
+  const body = document.querySelector('.flex');
+  const child = e.target.parentNode;
+  if (child && myLibrary.length !== 0) {
+    body.removeChild(child);
+    const index = e.srcElement.attributes['1'].value;
+    delete myLibrary[index];
+  }
+}
+
+function render() {
+  const div = document.createElement('div');
+  const title = document.createElement('h3');
+  const author = document.createElement('p');
+  const pages = document.createElement('p');
+  const button = document.createElement('button');
+  button.addEventListener('click', (e) => {
+    switchStatus(e);
+    e.preventDefault();
+  });
+  const remove = document.createElement('button');
+  remove.addEventListener('click', (e) => {
+    removeBook(e);
+    e.preventDefault();
+  });
+  const body = document.querySelector('.flex');
+  const data = JSON.parse(localStorage.getItem('items'));
+  data.forEach((item, i) => {
```

Comment:
You might want to create the elements for each book inside the loop, otherwise, you are overwriting the same element instead of creating a new one for each book.